### ISSUE TITLE
exclusive Flag problems / resolvconf location

### ISSUE THREAD

**Issue by andywuest** [masterkorp/openvpn-update-resolv-conf#8]
Hi,

i have finally successully used your problem. However i encountered one problem:
you have lately added the exclusive flag, which does not work for me, since i am on a old
distro. As you can see the flag was added in ubuntu in 15.10. however i am still on 14.04 LTS. 
Maybe you can make this more dynamic via calling (lsb_release -c) to check the version.
Unfortunately the resolvconf itself does not show a version, so it is not so easy to find out
if the resolvconf supports the -x or not.

http://manpages.ubuntu.com/manpages/trusty/en/man8/resolvconf.8.html
http://manpages.ubuntu.com/manpages/wily/en/man8/resolvconf.8.html

Another thing i noticed is this.
## You might need to set the path manually here, i.e.
RESOLVCONF=/sbin/resolvconf

why not use
RESOLVCONF=$(type -p resolvconf)
this will find the path to the resolvconf by itself.

**Comment by GitHorst** [comment]
I encountered exactly the same issues on Mint 17.3 (i.e. LTS 14.04).
Basically I cannot use it, because I don't have -x flag available but without the flag (i.e. if i remove it from the line in the script), it does not update the resolv.conf correctly and thus i still cannot access the local servers at work through the vpn from home.

**Comment by voldemorte** [comment]
I'm on a Fedora 24 machine and I'm facing the similar problem.
The resolvconf binary is not available and the command type -p resolvconf shows up empty.

**Comment by masterkorp** [comment]
You are right; this can easly resolved.with this and a fallback to `/sbin/resolvconf` If not binary is found.

**Comment by markus-daniel** [comment]
On Fedora 26 there is no resolvconf binary installed nor available.
I assume I have to use an other way to update /etc/resolv.conf

**Comment by PropertyOfTuckerSystems** [comment]
@markus-daniel just download and build it, e.g. https://gist.github.com/uzantonomon/5489a4d9fe4caf813f45bf4bde3a1ffe
